spec/system/oauth_spec.rb

```diff
@@ -115,6 +115,8 @@
           subject
 
           within '.form-container .flash-message' do
+            # FIXME: Replace with doorkeeper.errors.messages.invalid_code_challenge_method.one for Doorkeeper > 5.8.0
```

Comment:
I think I get this, but just to confim -- this would be fine to merge as-is right now (with comment in place), but then at whatever point a new release is tagged on doorkeeper side and we update, we can switch this i18n key?

Reply:
Yeah, we swap the key, and it'll work. So some version > 5.8.0 will use that new key.